test/validation/validators.test.js

```diff
@@ -17,6 +19,27 @@ describe('validators', () => {
 		assert.throws(() => validatePositiveNumber(NaN), /Error/);
 	});
 
+	it('should validate number', () => {
+		assert.strictEqual(validateNumber(0), 0);
```

Comment:
negative value should be present in the test, that is the purpose of this method, add it to test

Reply:
added